doc/source/whatsnew/v0.19.0.txt

```diff
@@ -432,6 +432,7 @@ Deprecations
 - ``buffer_lines`` has been deprecated in ``pd.read_csv()`` and will be removed in a future version (:issue:`13360`)
 - ``as_recarray`` has been deprecated in ``pd.read_csv()`` and will be removed in a future version (:issue:`13373`)
 - top-level ``pd.ordered_merge()`` has been renamed to ``pd.merge_ordered()`` and the original name will be removed in a future version (:issue:`13358`)
+- Removal of the legacy time rules (offset aliases), deprecated since 0.14.0 (this has been alias since 0.8.0) (GH13590).
```

Comment:
put in a top-level separate section (prior deprecation removal - look how I did this for 0.18.0)